tests/foreman/api/test_contentaccess.py

```diff
@@ -0,0 +1,75 @@
+# -*- encoding: utf-8 -*-
+"""Test for Content Access (Golden Ticket) API
+
+:Requirement: Content Access
+
+:CaseLevel: Acceptance
+
+:CaseComponent: API
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.decorators import (
+    run_only_on,
+    stubbed,
+    tier2,
+)
+from robottelo.test import APITestCase
+
+
+class ContentAccessTestCase(APITestCase):
+    """Content Access API tests."""
+
+    @classmethod
+    def setUp(self):
+        """Setup must ensure there is an Org with Golden Ticket enabled.
+
+        SQL::
+
+            UPDATE
+                 cp_owner
+            SET
+                 content_access_mode = 'org_environment',
+                 content_access_mode_list='entitlement,org_environment'
+            WHERE account='{org.label}';
+
+        :steps:
+
+            1. Create a new organization.
+            2. Create a Product and CV for org.
+            3. Add a repository pointing to a real repo which requires a
+               RedHat subscription to access.
+            4. Create Content Host and assign that gated repos to it.
+            5. Create Host with no attached subscriptions.
+            6. Access candlepin PG database and execute SQL to activate
+               the Golden Ticket.
```

Comment:
@sghai I thought it would be better if we can setup a simgle organization before those tests run, maybe as part of automation_tools so we start testing with everything in place but I dont know if we have a `master setup`